tests/attested_tls/oe_enclave/enc/public.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Do you need this file?

Reply:
please check the above one

Reply:
Confirmed w/ Xuejun, public.pem can be removed.